README.md

```diff
@@ -25,7 +25,10 @@
 
 ## Experiências profissionais
 
-### Avec - Front-end Engineer - Junho 2019 / Atual
+### Creditas - Software Engineer - Dezembro 2019 / Atual
```

Comment:
Change to Novembro 2019 and Senior Software Engineer please

Reply:
Fixed!